test/e2e/networkpolicy_test.go

```diff
@@ -52,34 +52,43 @@ func TestNetworkPolicy(t *testing.T) {
 	defer teardownTest(t, data)
```

Comment:
Would there be duplicate logs then? One in `exportLogsForSubtest` and one in `teardownTest exportLogs`?

Reply:
Yes, but it's already this case even without this change. For example, when two tests failed and there is no restart in-between, the logs collected by the latter one will contain the ones collected by the former. Since the logs are just used for debugging, I feel their redundancy may be not that important. To reduce the redundancy, the simplest way I can think of is adding a bool to `teardownTest` to indicate whether it should try to collect logs or not and remember to not set it for tests whose subtests have already collected logs.

Reply:
That makes sense, I had the question because `teardownTest` is indeed used by many other tests not requiring subtest logs, so I'm convinced it's fine if only used for debug directory.